plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/TestProjectCreator.java

```diff
@@ -162,8 +163,10 @@ private void addFacets() throws CoreException {
     }
     facetUtil.install(null);
 
-    // App Engine runtime is added via a Job, so wait.
-    ProjectUtils.waitForProjects(getProject());
+    if (facetedProject.hasProjectFacet(AppEngineStandardFacet.FACET)) {
```

Comment:
Not needed for flex. For standard, we add the App Engine runtime (which doesn't exist for flex) via a job, so that is why we should wait here.